SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -441,33 +444,61 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
         su2double kPlus = FrictionVel*Roughness_Height*density/laminar_viscosity;
 
         su2double S_R= 0.0;
+ 
+        su2double solution[2];
+        /*--- Modify the omega and k to account for a rough wall. ---*/
+
         /*--- Reference 1 original Wilcox (1998) ---*/
-        /*if (kPlus <= 25)
+        if (roughsstParsedOptions.wilcox1998) {
+          if (kPlus <= 25)
             S_R = (50/(kPlus+EPS))*(50/(kPlus+EPS));
           else
-            S_R = 100/(kPlus+EPS);*/
-
+          S_R = 100/(kPlus+EPS);
+          
+          solution[0] = 0.0;
+          solution[1] = FrictionVel*FrictionVel*S_R/(laminar_viscosity/density);          
+        } 
+        else if (roughsstParsedOptions.wilcox2006) {
         /*--- Reference 2 from D.C. Wilcox Turbulence Modeling for CFD (2006) ---*/
-        if (kPlus <= 5)
-          S_R = (200/(kPlus+EPS))*(200/(kPlus+EPS));
-        else
-          S_R = 100/(kPlus+EPS) + ((200/(kPlus+EPS))*(200/(kPlus+EPS)) - 100/(kPlus+EPS))*exp(5-kPlus);
-
-        /*--- Modify the omega to account for a rough wall. ---*/
-        su2double solution[2];
-        solution[0] = 0.0;
-        solution[1] = FrictionVel*FrictionVel*S_R/(laminar_viscosity/density);
+          if (kPlus <= 5)
+            S_R = (200/(kPlus+EPS))*(200/(kPlus+EPS));
+          else
+            S_R = 100/(kPlus+EPS) + ((200/(kPlus+EPS))*(200/(kPlus+EPS)) - 100/(kPlus+EPS))*exp(5-kPlus);
```

Comment:
Use the pow function please
```suggestion
            S_R = pow(200/(kPlus+EPS), 2);
          else
            S_R = 100/(kPlus+EPS) + (pow(200/(kPlus+EPS), 2) - 100/(kPlus+EPS))*exp(5-kPlus);
```